<?xml version="1.0"?>

<launch>
    <node pkg="foobar" type="foobar_node" name="foobar" output="screen">
        <param name="plane_tolerance" value="0.1"/>
        <param name="bar_tolerance" value="0.15"/>
        <param name="cluster_tolerance" value="0.01"/>
        <param name="bar_width" value="0.13"/>
        <param name="min_points" value="500"/>
        <param name="bar_length" value="0.43"/>
        <param name="input_topic" value="/pacman_vision/processed_scene"/>
        <param name="reference_frame" value="/camera_rgb_optical_frame"/>
    </node>
</launch>
